lib/config-generator.js

```diff
@@ -81,7 +81,7 @@ class ConfigGenerator {
         };
 
         if (this.webpackConfig.useVueLoader) {
-            config.resolve.alias['vue$'] = 'vue/dist/vue.esm.js';
+            config.resolve.alias['vue$'] = 'vue/dist/vue.common.js';
```

Comment:
this should switch between `esm` and `common` based on `this.webpackConfig.useTypeScriptLoader`, to avoid breaking the case of typescript